test/integration/node-specific/bson-options/useBigInt64.test.ts

```diff
@@ -0,0 +1,234 @@
+import { expect } from 'chai';
```

Comment:
this file should use snake case (`use_bigint_64.test.ts`) or something similar to follow our existing filenaming convention.